Content.Shared/Magic/SharedMagicSystem.cs

```diff
@@ -501,6 +509,59 @@ private void OnChargeSpell(ChargeSpellEvent ev)
         _gunSystem.UpdateBasicEntityAmmoCount(wand.Value, basicAmmoComp.Count.Value + ev.Charge, basicAmmoComp);
     }
     // End Charge Spells
+    #endregion
+    #region Global Spells
+
+    private void OnRandomGlobalSpawnSpell(RandomGlobalSpawnSpellEvent ev)
+    {
+        if (!_net.IsServer)
+            return;
+
+        if (ev.Handled || !PassesSpellPrerequisites(ev.Action, ev.Performer))
+            return;
+
+        if (ev.Spawns is not { } spawns)
+            return;
```

Comment:
this can all be condensed down to:

```suggestion
    {
        if (!_net.IsServer || ev.Handled || !PassesSpellPrerequisites(ev.Action, ev.Performer) || ev.Spawns is not { } spawns)
            return;
```